stix_shifter_modules/secretserver/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,78 @@
+{
+  "username": [
+    {
+    "key": "user-account.user_id"
+  },
+    {
+      "key": "x-ibm-finding.user_name",
```

Comment:
This could be `x-secretserver-finding` or a reference from the finding object to the user-account object.